rise/static/rise.yaml

```diff
@@ -0,0 +1,216 @@
+#
+# NOTE that the defaults defined here are essentially NOT used
+# in the code, but if only for documentation purposes it is important
+# that they reflect the defaults that are hardwired in main.js
+#
+Type: Jupyter Notebook Extension
+Compatibility: jupyter-4.3, jupyter-notebook-5.2
+Name: RISE - Slideshow with reveal.js
+Main: main.js
+Description: "Run notebook as a slideshow thanks to reveal.js"
+Link: configurator.md
+Icon: rise.png
+Parameters:
+#
+- name: rise.autolaunch
+  description: >
+     <code>autolaunch</code>: autorun slideshow upon opening if set; active only
+     on notebooks that have a 'livereveal' or 'rise' section in their
```

Comment:
Well, it will be active also with the configurator, not just the metadata... I think it is confusing as is written.

Reply:
This was related to the explanation above; I will add a few more words to try and make it clearer here